apps/backoffice-v2/src/pages/TransactionMonitoringAlerts/components/AlertsHeader/AlertsHeader.tsx

```diff
@@ -28,6 +36,53 @@ export const AlertsHeader: FunctionComponent<{
     },
     [mutateAssignAlerts, selected],
   );
+  const onMutateAlertsDecision: ComponentProps<typeof AlertsDecisionDropdown>['onDecisionSelect'] =
+    useCallback(
+      decision => () => {
+        const screamingSnakeDecision = toScreamingSnakeCase(decision);
+
+        mutateAlertsDecision({
+          decision: alertDecisionToState[screamingSnakeDecision],
+          alertIds: Object.keys(selected ?? {}),
+        });
+      },
+      [mutateAlertsDecision, selected],
+    );
+  const decisions = [
+    ...['Revert Decision' as const, ...AlertStates]
+      .map(state => {
+        const screamingSnakeState = toScreamingSnakeCase(state);
+
+        return alertStateToDecision[screamingSnakeState as keyof typeof alertStateToDecision];
+      })
+      .filter(Boolean)
+      .map(decision => ({
+        id: decision,
+        value: (
+          <>
+            {lowerCase(decision) === 'reject' && (
```

Comment:
use a constant

Reply:
Can you elaborate or use the `suggestion` feature?

Reply:
edited my comment

Reply:
Extract to a constant all uses of state names like 'reject', 'not suspicious', etc.